monero-outreach-docs/translations/fr/archive/dandelion_fr.md

```diff
@@ -0,0 +1,67 @@
+# Dandelion++ pour Monero
+
+**"Dandelion++ empêche les mauvais acteurs d'associer votre adresse IP à vos transactions Monero." - 2 mars, 2020**
+
+Monero est soutenu par son réseau de communication pair à pair (P2P). Les ordinateurs qui sous-tendent le réseau par leur connaissance de la chaîne de blocs — appelés noeuds — partagent l'information qui rend l'utilisation de Monero possible: les adresses de nœuds, l'historique de la chaîne de blocs, les nouveaux blocs minés, et les nouvelles transactions à ajouter aux blocs. Les nœuds sont toutefois identifiées selon leur adresse de Protocole Internet ("Internet Protocol" — IP — en anglais) exposant le réseau au risque d'associations entre les adresses IP et les transactions, qui provoqueraient une perte de l'anonymité des transaction et de l'information qu'elles renferment. Dandelion++ est une méthode pour cacher cette connexion, planifiée pour une prochaine mise à jour du logiciel de Monero. Cet article explique le fonctionnement de Dandelion++ et comment les usagers de Monero en bénéficieront.
+
+### Le Réseau P2P de monero
+
+Monero possède deux éléments conceptuels: 1) un réseau P2P et 2) des applications qui s'exécutent sur ce réseau. Les applications gèrent les adresses, les clés et les transactions, tandis que le réseau organise l'information et s'assure de la transmission des données. Pour bien comprendre Dandelion++, il est pertinent de considérer la structure du réseau P2P ainsi que la manière dont les portefeuilles s'en servent pour communiquer les transactions.
+
+Sa structure comprend des milliers de nœuds répartis de par la planète — des dizaines de milliers si on compte les nœuds passifs — chacun connecté, via Internet, habituellement à un petit groupe d'autres nœuds appelés pairs [1]. Les nœuds communiquent avec leurs pairs en utilisant le même protocole TCP ("Transmission Control Protocol" - Protocole de Contrôle de Transmission) utilisé par les fureteurs et les serveurs Web. Les pairs d'un nœud paraissent aléatoires et n'ont pas besoin de rapprochement géographique. Un nombre commun de pairs est huit, mais certains nœuds en ont des centaines. Chaque nœud trouve ses pairs en envoyant d'abord une requête à un nœud maître pour sa liste de pairs, pour ensuite envoyer la même requête aux pairs de cette liste, et ainsi de suite. Le résultat est un réseau de plusieurs ordinateurs, chacun communicant par TCP avec au mieux une visibilité partielle et limitée du réseau entier.
```

Comment:
fureteurs >>> navigateurs  

Reason: reviewer choice, but this is certainly due to a regional difference. People native from France - as opposed to e.g Quebec - might have never heard the term 'fureteur'

Reply:
Integrated.